### DISCUSSION TITLE
Unable to install Copilot4Eclipse from Marketplace

### DISCUSSION THREAD

**Discussion by chintankachhi** [Genuitec/Copilot4Eclipse-community#10] [Help] [answered]
I am trying to install the github copilot plugin for Eclipse but running into issues while trying to install it from the Eclipse Marketplace. I am using Eclipse version: 2023-12 (4.30.0), java 17. Please let me know how I can proceed past this error:

![image](https://github.com/user-attachments/assets/4d912e7b-4e36-4245-bf75-d0ae975396b5)

**Comment by wayneparrott** [comment, marked as answer]
Hi,
Please see this technical note that describes a quick configuration change to enable your system to find additional dependencies required by Copilot4Eclipse: https://www.genuitec.com/products/copilot4eclipse/docs/resources/technotes/configure-discovery-site
Let us know if this resolves your installation issue.
W
